app/src/main/res/values-ro/strings.xml

```diff
@@ -31,7 +31,7 @@ Acest lucru este cauzat de bug-uri in Android sau in software-ul pus la dispozit
     <string name="msg_revoked">NetGuard a fost dezactivat</string>
     <string name="msg_completed">Setari aplicate</string>
     <string name="msg_vpn">NetGuard foloseste o conexiune locala VPN ca o \"gaura neagra\" pentru traficul de internet blocat, din acest motiv la urmatoarea fereastra de dialog trebuie sa acceptati crearea conexiunii VPN.</string>
-    <string name="msg_try">Try NetGuard</string>
+    <string name="msg_try">Incearca NetGuard, firewall ce nu necesita root</string>
```

Comment:
The e-mail the invitee receives contains already part of the Play store description, so '*firewall ce nu necesita root*' seems unnecessary to me.